Do not look for OpenSSL in /usr/local/opt/openssl.
Do not assume the MacPorts prefix is /opt/local.
https://github.com/VirusTotal/yara/issues/1160
--- configure.ac.orig	2022-03-09 03:09:48.000000000 -0600
+++ configure.ac	2022-04-14 00:20:26.000000000 -0500
@@ -43,11 +43,7 @@
 esac
 
 case $host_os in
-    darwin*) CFLAGS="$CFLAGS -I/opt/local/include"
-             # Starting with Mac OS X 10.11 (El Capitan) the OpenSSL headers
-             # are in /usr/local/opt/openssl/include
-             CFLAGS="$CFLAGS -DUSE_MACH_PROC -I/usr/local/opt/openssl/include"
-             LDFLAGS="$LDFLAGS -L/usr/local/opt/openssl/lib"
+    darwin*) CFLAGS="$CFLAGS -DUSE_MACH_PROC"
              posix=true
              proc_interface=mach
              jemalloc_prefix=je_ ;;
